### PR TITLE
DM-33836: ATWhiteLight: add v2 directory for salobj 7

### PR DESCRIPTION
**Pull request by r-owen**

### CODE REVIEW

File: ATWhiteLight/v2/_summit.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+lamp:
+  identifier: 140.252.33.160
+chiller:
+  host: 140.252.33.70
```
Comment (patrickingraham) [inline ATWhiteLight/v2/_summit.yaml]: We should use DNS names here.
lanToSerialConverterChiller.tu.lsst.org

The labjack doesn't have DNS name yet...

File: ATWhiteLight/v2/_summit.yaml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+lamp:
```
Comment (patrickingraham) [inline ATWhiteLight/v2/_summit.yaml]: this should be `_tucson.yaml` and not summit.

File: ATWhiteLight/v2/_summit.yaml (hunk 1 above)
Comment (patrickingraham) [reply ATWhiteLight/v2/_summit.yaml]: You need a port as well, with is 4001 for this adapter.

**Review by patrickingraham — COMMENTED** [review]
Thanks. Looks nice.
I made a few comments.

File: ATWhiteLight/v2/_summit.yaml (hunk 1 above)
Comment (r-owen) [reply ATWhiteLight/v2/_summit.yaml]: Yes. That is set in _init.yaml. I assume all sites will use that port, but if not, then it certainly belongs in the site-specific file. I will use host names when I have them.

File: ATWhiteLight/v2/_summit.yaml (hunk 2 above)
Comment (r-owen) [reply ATWhiteLight/v2/_summit.yaml]: Good point. I changed it. I wish I could have a summit file as well, but it's not much use until we have the host names.

**Review by patrickingraham — APPROVED** [review]
Looks good. thanks

**Pull request merged**